mars/similarity_calculation.py

```diff
@@ -1,16 +1,25 @@
 from collections import defaultdict
-from typing import Dict, List
+from typing import Dict, List, Union
 
 import numpy as np
 from tqdm import tqdm
 
 from mars import db, logging, sentence_embeddings
-from mars.db import db_fields
+from mars.db import collections, db_fields
+from mars.db.db import fetch_batches_until_empty
+from mars.keyword_topic_model import KeywordTopicModel
+from mars.models_training.datasets import DocumentLevelDataset, targets
+
+logger = logging.new_logger(__name__)
 
 
 def similarity(sent_embedding: np.ndarray, query_embedding: np.ndarray) -> float:
     """Calculates similarity between single sentence and query (both provided as embeddings)."""
-    return np.matmul(np.array(sent_embedding), np.transpose(query_embedding))
+    return (
+        1
+        + np.matmul(np.array(sent_embedding), np.transpose(query_embedding))
+        / (np.linalg.norm(sent_embedding) * np.linalg.norm(query_embedding))
+    ) / 2
```

Comment:
@Gizzio zrobisz do tego jakiś test? Nie musi być teraz, ale generalnie to chyba ważny komponent i warto testować żeby się nie wykrzaczył